### ISSUE TITLE
why use cross entropy loss and not BCE loss? 

### ISSUE THREAD

**Issue by mshooter** [NaJaeMin92/pytorch_DANN#6]

**Comment by NaJaeMin92** [comment]
Hi @mshooter 

Of course, since the `Discriminator` just conducts binary classification, you can also implement it using `sigmoid` and `BCE loss` instead of insisting on `cross-entropy loss`. Both BCE and Cross-entropy loss would work well with no significant differences.
